src/broadcast.jl

```diff
@@ -37,6 +37,13 @@ for f in libdevice
   @eval cufunc(::typeof(Base.$f)) = CUDAnative.$f
 end
 
+#broadcast ^
+CUDAnative.pow(x::F, y::Int64) where F = CUDAnative.pow(x, Int32(y))
```

Comment:
```suggestion
CUDAnative.pow(x::Union{Float32, Float64}, y::Int64) = CUDAnative.pow(x, Int32(y))
```

this definition should probably live in `CUDAnative` itself.